src/plugin.py

```diff
@@ -400,6 +408,10 @@ async def get_friends(self):
 
     async def launch_game(self, game_id):
         if is_uri_handler_installed("origin2"):
+            entitlement = self._entitlement_cache.get(game_id, None)
```

Comment:
None is default value of dict's `get` method 
```suggestion
            entitlement = self._entitlement_cache.get(game_id)
```